### ISSUE TITLE
AnalyticalCard: Accesibility properties not available hence not focusable by Tab key

### ISSUE THREAD

**Issue by sparkypro** [SAP/ui5-webcomponents-react#3149]
**Describe the bug**
Analytical Card UI component is not reachable by Keyboard Tab key specifically. There is no accessibility options available for analytical card ui.

**UI Link**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticalcard--default-story

**To Reproduce**
1. Go to UI Link
2. Use Tab key from keyboard to focus on the Analytical Card UI.

**Expected behavior**
Analytical card ui must be reachable by Tab key from keyboard, and must have accessibility properties to read out child elements

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.3.1
`@ui5/webcomponents-react` version:0.23.2
Operating System: Windows 11
Browser: Chrome
